flexget/plugins/sites/torrentday.py

```diff
@@ -123,30 +123,39 @@ def search(self, task, entry, config=None):
             categories = [categories]
         # If there are any text categories, turn them into their id number
         categories = [c if isinstance(c, int) else CATEGORIES[c] for c in categories]
-        params = { 'cata': 'yes', 'c%s' % ','.join(str(c) for c in categories): 1, 'clear-new': 1}
+        params = { 'cata': 'yes', 'c{}'.format(','.join(str(c) for c in categories)): 1, 'clear-new': 1}
         entries = set()
         for search_string in entry.get('search_strings', [entry['title']]):
 
-            url = 'https://www.torrentday.com/browse.php'
-            params['search'] = normalize_unicode(search_string).replace(':', '')
+            url = 'https://www.torrentday.com/t'
+            params['q'] = normalize_unicode(search_string).replace(':', '')
             cookies = { 'uid': config['uid'], 'pass': config['passkey'], '__cfduid': config['cfduid'] }
 
             try:
-                page = requests.get(url, params=params, cookies=cookies).content
+                page = requests.get(url, params=params, cookies=cookies)
             except RequestException as e:
-                raise PluginError('Could not connect to torrentday: %s' % e)
+                raise PluginError('Could not connect to torrentday: {}'.format(e))
 
-            soup = get_soup(page)
-
-            for tr in soup.find_all('tr', { 'class': 'browse' }):
+            soup = get_soup(page.content)
+            form = soup.find('form', id='torrents')
+            table = form.find('table', { 'id': 'torrentTable' })
+            # the first row is the header so skip it
+            for tr in table.find_all('tr')[1:-1]:
```

Comment:
Is this meant to skip the last row too? If you just want to skip the first row, you probably want `[1:]`

Reply:
Yeah, that is what I originally did, but it caused a runtime error. I was so confused by that.